build-parent/pom.xml

```diff
@@ -111,6 +111,9 @@
     <!-- MP Config implementation -->
     <version.wildfly-microprofile-config>1.2.1</version.wildfly-microprofile-config>
 
+    <!-- SmallRye -->
```

Comment:
I'll add a 'hold' label till the smallrye-metrics is released

Reply:
:+1: